lib/brains/tests/samples/customer-error.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<api-error-response>
+  <errors>
+    <errors type="array"/>
+    <customer>
+      <errors type="array">
+        <error>
+          <code>91610</code>
+          <attribute type="symbol">id</attribute>
+          <message>Customer ID is invalid (use only letters, numbers, '-', and '_').</message>
+        </error>
+      </errors>
+    </customer>
+  </errors>
+  <params>
+    <customer>
+      <id>sdfsdf:sdf:</id>
+    </customer>
+  </params>
+  <message>Customer ID is invalid (use only letters, numbers, '-', and '_').</message>
+</api-error-response>
```

Comment:
This feels like a really heavy fixture when a mock patch would probably only require something like:
````
result = mock.Mock()
result.is_success = False
result.message = 'Customer ID is invalid...'
patched_customer.create.return_value = result
````